src/test/java/com/epam/tam/unittests/CalculatorTest.java

```diff
@@ -0,0 +1,99 @@
+package com.epam.tam.unittests;
+
+import com.epam.tat.module4.Calculator;
+
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class CalculatorTest {
+
+	private Calculator calculator;
+	private static final double DELTA = 0.01;
+	
+	@BeforeClass
+	public void setUp(){
+		calculator = new Calculator();
+	}
+	
+	@Test(expectedExceptions = IllegalArgumentException.class, dependsOnGroups = {"basic"})
+	public void divideByZeroThrowsException() {
+		calculator.div(2, 0);
+	}
+	
+	@Test(dataProvider = "dataForSum", groups = {"basic"})
+	public void testSum(long first, long second, long expected) {
+		long result = calculator.sum(first, second);
+		Assert.assertEquals(result, expected);
+	}
+	
+	@DataProvider(name = "dataForSum")
+	public Object[][] dataForSum() {
+		return new Object[][] {
+				{0, 0, 0},
+				{-1, 0, -1},
+				{-10, 10, 0},
+				{2, 2, 4}
+		};	
+	}
+	
+	@Test(dataProvider = "dataForMult", groups = {"basic"})
+	public void testMult(long first, long second, long expected) {
+		long result = calculator.mult(first, second);
+		Assert.assertEquals(result, expected);
+	}
+	
+	@DataProvider(name = "dataForMult")
+	public Object[][] dataForMult() {
+		return new Object[][] {
+				{0, 0, 0},
+				{-1, 0, 0},
+				{-10, 10, -100},
+				{2, 2, 4}
+		};	
+	}
+	
+	@Test(expectedExceptions = IllegalArgumentException.class)
+	public void sumTwoMaxLongThrowsException() {
+		calculator.sum(Long.MAX_VALUE, Long.MAX_VALUE);
+	}
+	
+	@Test(groups = {"basic"})
+	public void testPositive() {
+		boolean result = calculator.isPositive(7);
```

Comment:
I thought it is just more convenient for debuggin in case debugging is needed, am i mistaken?